.github/workflows/lint.yml

```diff
@@ -0,0 +1,20 @@
+name: Lint
+on:
+  push:
+    branches: [main]
+  pull_request:
+    branches: [main]
+jobs:
+  lint:
+    name: Lint Files
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
```

Comment:
Give it a name lol

Reply:
The default name is "Run actions/checkout@v2", which I thought was fine.

Should I change it to something like "Checkout the repository"?

Reply:
Yes!